README.md

```diff
@@ -33,45 +33,42 @@ If you have any questions, comments, insights, praises, or doubts about these do
 
 - [Limitations](#limitations) Describes the current limitations of Ray on Golem
 - [Quickstart](#quickstart) Drives you through copy and paste installation and execution of the example Ray app on example Ray on Golem cluster
-- [Contributing](#contributing) Offers advice on building Golem images for Ray on Golem development purposes
-
 
 # Limitations
 
-Current version is `pre-alpha` which means the happy path is working on Ubuntu on the Golem test network.
+Current version is `pre-alpha` which means the happy path is working on **Ubuntu** on the Golem test network.
 We have tested Ray on Golem on Ubuntu and on WSL, but it should work on other Linux distributions. At the moment, we don't support MacOS or bare Windows.
  
 We use this version to show the direction and get feedback.
 
-There is one Ray on Golem image. It contains `ray 2.3` and `python 3.10`.
+There is one Ray on Golem image. It contains `ray 2.7.1` and `python 3.10.13`.
 It should work with any combination of local ray & python versions. Please let us know if you have any troubles because of that (on [`#Ray on Golem` discord channel](https://chat.golem.network/))
 
 The images include only basic libraries, if you need any dependencies, 
 you can use `pip` via [cluster yaml `initialization_commands`](https://golem-docs-git-mateusz-ray-on-golem-pre-alpha-golem.vercel.app/docs/creators/ray/cluster-yaml-reference#initializationcommands)
 
-We have tested Ray on Golem on Ubuntu and on WSL. It should work on MacOS and shouldn't on bare Windows.
-
-# QuickStart
+# Quickstart
 
 This [quickstart](https://docs.golem.network/docs/creators/ray/quickstart) shows you how to set Ray and Ray on Golem up, start your cluster, test it, and then stop it.
 It limits the explanation to the bare minimum - if you are looking for more details jump to [setup tutorial](https://docs.golem.network/docs/creators/ray/setup-tutorial)
 
-We recommend creating a new directory and a clean Python virtual environment before you proceed. This avoids cluttering your system installation with unnecessary packages.
 
 ## Install software
 
+**Note:** We recommend creating a new directory and a clean Python virtual environment before you proceed. This avoids cluttering your system installation with unnecessary packages.
+
 The first step is installing Ray on Golem. It will install Ray as a dependency.
 
 ```bash
-# install ray-on-golem and ray (recommended within a clean venv)
+# install ray-on-golem & ray (recommended within a clean virtual environment)
```

Comment:
what's wrong with `and` ?

Reply:
nothing, is it for consistency with changes in the docs, we had a couple lines like that there, had to pick one